css and xpath.txt

```diff
@@ -0,0 +1,50 @@
+CSS project 
+
+1. Page: /text-box. Write a selector that will return the ‘Full Name’ textbox using the ID attribute.
+Hint: This element has the ‘input’ tag.
+
+$("[id=userName]")
+
+2. Page: /buttons. Write a selector that will return all 3 buttons shown below
+
+$("button[class~=btn-primary]")
+
+3. Page: /buttons. Write a selector that will return the span element in the footer.
+$("#app > footer > span")
+
+4. Page: /menu. Write a selector that will return the 3 list item elements in the unordered list
+$("#nav > li")
+
+5. Page: /select-menu. Write a selector that will get the option element where value = 4.
+$("option[value='4']")
+
+6.Page: /slider. Write the JavaScript to get the value for the element with id=sliderValue.

```

Comment:
This doesn't get the value, it gets the element
document.getElementsByClassName('range-slider range-slider--primary')[0].value